elastic_connect/data_types/base.py

```diff
@@ -0,0 +1,53 @@
+from abc import ABC
+import datetime
+import dateutil
+
+class BaseDataType(ABC):
+
+    def __init__(self, name):
+        self.name = name
+
+    def from_dict(self, value):
```

Comment:
v ramci data typu pouzivat jen metody:

`from_es` a `from_python`: at dostanu cokoliv, tak do chlivecku objektu ulozim to co tam ma byt (nebo upadnu). V zakladu budou obe fungovat totozne, pro joiny se muzou lisit (ale nemusi, muzu chtit povolit udelat User.child = 12, aby mi to do User.child_id dalo jen idcko 12)

`to_es`: ktera se bude pouzivat pro serializaci objektu do elasticsearche